pyproject.toml

```diff
@@ -95,6 +95,10 @@ select = [
 
 ignore = [
     "F403", # 'from module import *' used; unable to detect undefined names
+    "E501", # Line too long
+    "PLR0912", # Too many branches
+    "PLR0913", # Too many arguments in function definition
+    "PLR0915", # Too many statements
```

Comment:
Maybe we can keep these checks and ignore them locally when needed?

Reply:
I will use the same one we have in nomad-FAIR repo...